authors/apps/articles/response_messages.py

```diff
@@ -0,0 +1,7 @@
+ERROR_MESSAGES = {
+    "forbidden" : "cannot report own article",
+    "not found" : "no report made by this user",
+    "article" : "article not found",
+    "report" : "report not found",
+    "rating" : "rating not found"
```

Comment:
You can have the `not found messages as one, eg. ```"record_not_found":  "{} not found"```. You can then use `.format()` when accessing the message, passing the appropriate val, eg ```ERROR_MESSAGES["record_not_found"].format("article")``` or sth along these lines

Also add a blank line to the end of this file as per pep8

Reply:
done